scripts/dap-ami-bootstrap.sh

```diff
@@ -1,38 +1,42 @@
 #!/bin/bash
 set -e
 
-if ! docker images | grep conjur; then
-  echo "Loading Conjur appliance image into Docker"
-  gzip -df /home/core/conjur-appliance.tar.gz # Unzip the tar.gz into a tar
-  docker load -i /home/core/conjur-appliance.tar
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
 fi
 
 image_id=$(docker images -q)
-container_name='conjur-appliance'
+container_name='dap-appliance'
 
 docker rm -f ${container_name} > /dev/null 2>&1 || true
 
 mkdir -p /var/log/conjur
 mkdir -p /opt/conjur/backup
 
-echo "Creating Conjur container"
+echo "Creating DAP container"
 cid=$(docker create \
 --name ${container_name} \
 --privileged --restart always \
 --log-driver=journald \
 -v /var/log/conjur:/var/log/conjur \
 -v /opt/conjur/backup:/opt/conjur/backup \
```

Comment:
```suggestion
-v /opt/cyberark/dap/backup:/opt/conjur/backup:Z \
```

Reply:
Can we also mount:
```
    --volume /opt/cyberark/dap/configuration:/opt/cyberark/dap/configuration:Z \
    --volume /opt/cyberark/dap/security:/opt/cyberark/dap/security:Z \
    --volume /opt/cyberark/dap/seeds:/opt/cyberark/dap/seeds:Z \
```

Reply:
>`/opt/cyberark/dap/backup:/opt/conjur/backup`

Similar here, we should be careful renaming paths in the AMI only. That's going to be confusing when the paths don't match our docs anymore.

Reply:
This is done, all in addition to the existing as a middle-road option.